### ISSUE TITLE
function when-compiled not yet recognized

### ISSUE THREAD

**Issue by Reinhard-Prehofer** [uwol/cobol85parser#17]
Obviously the intrinsic function when-compiled is not yet implemented:

Statements like the following
```
000300             MOVE 'Compile-Datum: '      TO P190-TEXT (30:15)     15.04.98
000200             MOVE FUNCTION WHEN-COMPILED (7:2)                    22.09.97
000450                                         TO P190-TEXT (45:02)     15.04.98
000440             MOVE '.'                    TO P190-TEXT (47:01)     15.04.98
000200             MOVE FUNCTION WHEN-COMPILED (5:2)                    22.09.97
000300                                         TO P190-TEXT (48:02)     15.04.98
000440             MOVE '.'                    TO P190-TEXT (50:01)     15.04.98
000200             MOVE FUNCTION WHEN-COMPILED (1:4)                    22.09.97
000300                                         TO P190-TEXT (51:04)     15.04.98

```
lead to the following errors:
```
call to unknown data element FUNCTIONWHEN-COMPILED(7:2)
call to unknown data element FUNCTIONWHEN-COMPILED(5:2)
call to unknown data element FUNCTIONWHEN-COMPILED(1:4)
call to unknown data element FUNCTIONWHEN-COMPILED(9:2)
...
```

**Comment by Reinhard-Prehofer** [comment]
[WHENCOMP1.CBL.txt](https://github.com/uwol/cobol85parser/files/1181092/WHENCOMP1.CBL.txt)
A .CBL extension ist not allowed ! so I added a .txt suffix

**Comment by uwol** [comment]
Thanks! Should be fixed in 7b5cb5152eb52585c928162c1b87567fab11b10a.

The concept of function calls had been implemented in the grammar/AST and thus could be parsed, however has been missing in the semantic analyzer/ASG. Therefore I have added the class `FunctionCall` and extended the corresponding unit tests with the example provided by you.
